From 06d189ddd42aa7e8d26f1c2f72e4f2010bfd187b Mon Sep 17 00:00:00 2001
From: Jenny TC <jenny.tc@intel.com>
Date: Tue, 6 Jan 2015 08:49:57 +0530
Subject: [PATCH 1/1] pmic_ccsm: Set CDP input current as 1.5A

Set CDP input current as 1.5A to comply with BC1.2 spec

Change-Id: I80614d09b05b1790cc340b5f8b439481a05116f5
Signed-off-by: Jenny TC <jenny.tc@intel.com>
---
 drivers/power/intel_pmic_ccsm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index be98ac1..b6dd76f 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -48,7 +48,7 @@
 #define USB_WAKE_LOCK_TIMEOUT	(5 * HZ)
 
 #define USBINPUTICC100VAL	100
-#define HVDCP_POWER_CHRG_CURRENT 1500
+#define CDP_INPUT_CURRENT_LIMIT 1500
 #define HIGH_POWER_CHRG_CURRENT 2000
 #define LOW_POWER_CHRG_CURRENT 500
 
@@ -886,7 +886,7 @@ static void handle_internal_usbphy_notifications(int mask)
 			evt =  USB_EVENT_VBUS;
 		else
 			evt =  USB_EVENT_NONE;
-		cap.ma = HIGH_POWER_CHRG_CURRENT;
+		cap.ma = CDP_INPUT_CURRENT_LIMIT;
 		break;
 	case POWER_SUPPLY_CHARGER_TYPE_USB_DCP:
 	case POWER_SUPPLY_CHARGER_TYPE_SE1:
-- 
1.9.1

